config/exporter_config.yaml

```diff
@@ -8,44 +8,57 @@ bag_path: "/path/to/your.bag"
 output_dir: "/path/to/output_directory"
 
 # Storage format for the exported data. Options are "sqlite3" or "mcap"
-storage_id: "sqlite3"
+storage_id: "mcap"
 
 # List of topics to extract from the ROS2 bag
 topics:
   # Configuration for the point cloud topic
   - name: "/pointcloud_topic"   # Name of the topic to extract
     type: "PointCloud2"         # Data type of the topic
     sample_interval: 10         # Write one sample every 10 messages
+    topic_dir: "lidar"          # Output data in <output_dir>/<bag_name>/lidar
 
   # Configuration for the RGB image topic
   - name: "/rgb_image_topic"    # Name of the topic to extract
     type: "Image"               # Data type of the topic
     encoding: "rgb8"            # Encoding format for the image
     sample_interval: 5          # Write one sample every 5 messages
+    topic_dir: "cam/rgb"        # Output data in <output_dir>/<bag_name>/cam/rgb
+
+  # Configuration for the compressed image topic
+  - name: "/compressed_image_topic"  # Name of the topic to extract
+    type: "CompressedImage"          # Data type of the topic
+    sample_interval: 5               # Write one sample every 5 messages
+    topic_dir: "cam/rgb"             # Output data in <output_dir>/<bag_name>/cam/rgb
```

Comment:
Is this correct? Is the intention to put compressed/raw in the same directory (if we do end up extracting both for some reason?). Just because the readme is now setup to say that they are put in separate folders.
Thinking about it we may as well put raw/compressed in separate folders just in case for future proofing. At the end of the day the python script consuming this won't be much different if it is targeting `/cam/rgb` or `/cam/compressed`.
I propose the two folders should be `/cam/raw` and `/cam/compressed` depending on the image handler used.